<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-02-15T11:19:27 IST" name="Suite" finished-at="2022-02-15T11:20:11 IST" duration-ms="43712">
    <groups>
    </groups>
    <test started-at="2022-02-15T11:19:27 IST" name="Test1" finished-at="2022-02-15T11:19:41 IST" duration-ms="13728">
      <class name="e2EProject.FilterSearchTestCase">
        <test-method signature="BusCount()[pri:0, instance:e2EProject.FilterSearchTestCase@3e11f9e9]" started-at="2022-02-15T11:19:27 IST" name="BusCount" finished-at="2022-02-15T11:19:41 IST" duration-ms="13658" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BusCount -->
      </class> <!-- e2EProject.FilterSearchTestCase -->
    </test> <!-- Test1 -->
    <test started-at="2022-02-15T11:19:41 IST" name="Test2" finished-at="2022-02-15T11:19:53 IST" duration-ms="12155">
      <class name="e2EProject.HomePageTestCase">
        <test-method signature="pageinvok()[pri:0, instance:e2EProject.HomePageTestCase@2a265ea9]" started-at="2022-02-15T11:19:41 IST" name="pageinvok" finished-at="2022-02-15T11:19:53 IST" duration-ms="11772" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at e2EProject.HomePageTestCase.pageinvok(HomePageTestCase.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pageinvok -->
      </class> <!-- e2EProject.HomePageTestCase -->
    </test> <!-- Test2 -->
    <test started-at="2022-02-15T11:19:53 IST" name="Test3" finished-at="2022-02-15T11:20:11 IST" duration-ms="17792">
      <class name="e2EProject.BothDOJDORTestCase">
        <test-method is-config="true" signature="open()[pri:0, instance:e2EProject.BothDOJDORTestCase@449a4f23]" started-at="2022-02-15T11:19:17 IST" name="open" finished-at="2022-02-15T11:19:27 IST" duration-ms="9489" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- open -->
        <test-method is-config="true" signature="startup()[pri:0, instance:e2EProject.BothDOJDORTestCase@449a4f23]" started-at="2022-02-15T11:19:27 IST" name="startup" finished-at="2022-02-15T11:19:27 IST" duration-ms="410" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startup -->
        <test-method signature="DOJDOR()[pri:0, instance:e2EProject.BothDOJDORTestCase@449a4f23]" started-at="2022-02-15T11:19:53 IST" name="DOJDOR" finished-at="2022-02-15T11:20:11 IST" duration-ms="17786" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DOJDOR -->
        <test-method is-config="true" signature="flashout()[pri:0, instance:e2EProject.BothDOJDORTestCase@449a4f23]" started-at="2022-02-15T11:20:11 IST" name="flashout" finished-at="2022-02-15T11:20:12 IST" duration-ms="957" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flashout -->
      </class> <!-- e2EProject.BothDOJDORTestCase -->
    </test> <!-- Test3 -->
  </suite> <!-- Suite -->
</testng-results>
